cli/tests/testdata/chart1/values.yaml

```diff
@@ -3,17 +3,21 @@
 #    https://github.com/timescale/timescaledb-kubernetes/tree/master/charts/timescaledb-single
```

Comment:
Not for this PR, but these need to be refactored really really bad

Reply:
Can you elaborate on this? What needs to be changed here.

Reply:
Replied in DM :) 
This way is very tedious and error prone, we should just reference the helm chart directly and have only the changed value specified in the tests. Not to be done in this PR